client/ayon_resolve/api/lib.py

```diff
@@ -906,6 +908,32 @@ def _convert_resolve_list_type(resolve_list):
     return [resolve_list[i] for i in sorted(resolve_list.keys())]
 
 
+def get_clip_resolution_from_media_pool(timeline_item_data):
+    """Return the clip resolution from media pool data.
+
+    Args:
+        timeline_item_data (dict): Timeline item to investigate.
+
+    Returns:
+        resolution_info (dict): The parsed resolution data.
+    """
+    clip_item = timeline_item_data["clip"]["item"]
+    media_pool_item = clip_item.GetMediaPoolItem()
+    clip_properties = media_pool_item.GetClipProperty()
+
+    try:
+        width, height = clip_properties["Resolution"].split("x")
+    except (KeyError, ValueError):
+        width = height = None
+
+    try:
+        pixel_aspect = int(clip_properties["PAR"]) # Pixel Aspect Resolution
```

Comment:
PEP 8: E261 at least two spaces before inline comment
```suggestion
        pixel_aspect = int(clip_properties["PAR"])  # Pixel Aspect Resolution
```

Reply:
Also, what's the value of "PAR"? Because, isn't pixel aspect ratio usually a float?